packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { PaddedView } from '@apollosproject/ui-kit';
+
+import PrayerCard from './PrayerCard';
+
+const cardData = {
+  imageSource: {
+    uri: 'https://fillmurray.com/400/600',
+  },
+  name: 'Bill',
+  prayer:
+    'I’m alright. Nobody worry ’bout me. Why you got to gimme a fight? Can’t you just let it be?',
+  source: 'Anderson',
+};
+
+storiesOf('PrayerCard', module).add('Example', () => (
+  <PaddedView>
+    <PrayerCard
+      imageSource={cardData.imageSource}
+      name={cardData.name}
+      prayer={cardData.prayer}
```

Comment:
I think this is now `text` in the `PrayerCard` itself so the text of the prayer no longer displays in Storybook.
```suggestion
      text={cardData.text}
```